ADT7470.cpp

```diff
@@ -243,16 +243,16 @@ bool ADT7470::setPulsesPerRevolution(uint8_t idx, uint8_t val)
   uint8_t reg;
   _read(ADT7470_FAN_PPR_REGISTER, &reg);
   reg &= ~mask;
-  reg |= (val << (idx * 2));
+  reg |= ((val - 1) << (idx * 2));
   _write(ADT7470_FAN_PPR_REGISTER, reg);
   return true;
 }
 
 uint8_t ADT7470::getPulsesPerRevolution(uint8_t idx)
 {
   if (idx >= 4) return 0;
-  uint8_t reg = getReg8(ADT7470_FAN_PPR_REGISTER + idx);
-  return (reg >> (idx * 2)) & 0x03;
+  uint8_t reg = getReg8(ADT7470_FAN_PPR_REGISTER);
+  return ((reg >> (idx * 2)) & 0x03) + 1;
```

Comment:
OK, change is correct 
TODO: A note that the function returns 1..4  should be in the read,md and in the ADT7470.h file